src/py_ext_control_template/controller.py

```diff
@@ -0,0 +1,411 @@
+## Controller interface for drone toolbox in python
+## by Max Lodel, TU Delft, 2022
+
+import enum
+import numpy as np
+
+import rospy
+from drone_toolbox_ext_control_template.cfg import ControllerConfig
+
+from mavros_msgs.msg import PositionTarget, AttitudeTarget
+from mav_msgs.msg import RollPitchYawrateThrust
+from nav_msgs.msg import Odometry
+from geometry_msgs.msg import Pose
+
+from std_srvs.srv import Trigger
+
+from py_ext_control_template.custom_timer import CustomTimer
+
+from tf.transformations import quaternion_from_euler
+
+class ControlMethod(enum.Enum):
```

Comment:
A general comment: I think it's nice to stick to the PEP style suggestions by PyCharm in our Python code. Can you edit all files accordingly?

For example, this line has to be moved one line further to have two white lines before the class.